apps/vmq_server/src/vmq_info_cli.erl

```diff
@@ -0,0 +1,95 @@
+    clique:register_usage(["vmq-admin", "session", "list"], vmq_session_list_usage()).
+
+vmq_session_list_cmd() ->
+    Cmd = ["vmq-admin", "session", "list"],
+    KeySpecs = [],
+    ValidInfoItems = vmq_info:session_info_items(),
+    FlagSpecs = [{I, [{longname, atom_to_list(I)}]} || I <- [limit|ValidInfoItems]],
+    Callback = fun(_, [], Flags) ->
+                       Limit = proplists:get_value(limit, Flags, "100"),
+                       _ = list_to_integer(Limit),
+                       {Fields, Where} =
+                       case Flags of
+                           [] ->
+                               {["*"], []};
+                           _ ->
+                               lists:foldl(
+                                 fun({Flag, undefined}, {AccFields, AccWhere}) ->
+                                         {[atom_to_list(Flag)|AccFields], AccWhere};
+                                    ({Flag, Val}, {AccFields, AccWhere}) ->
+                                         {AccFields, [{atom_to_list(Flag), v(Val)}|AccWhere]}
+                                 end, {[],[]}, Flags)
+                       end,
+                       FFields =
+                       case Fields of
+                           [] ->
+                               "*";
+                           _ ->
+                               string:join(Fields, ",")
+                       end,
+                       WWhere =
+                       case Where of
+                           [] ->
+                               [];
+                           _ ->
+                               "WHERE " ++ lists:flatten(
+                                  string:join([[W, "=", V] || {W,V} <- Where], " AND "))
+                       end,
+                       QueryString = "SELECT " ++ FFields ++ " FROM sessions " ++ WWhere ++ " LIMIT " ++Limit,
+                       Table =
+                       vmq_ql_query_mgr:fold_query(
+                         fun(Row, Acc) ->
+                                 [maps:to_list(Row)|Acc]
+                         end, [], QueryString),
+                       [clique_status:table(Table)]
+               end,
+    clique:register_command(Cmd, KeySpecs, FlagSpecs, Callback).
+
+v("true" = V) -> V;
+v("false" = V) -> V;
+v(V) ->
+    try
+        _ = list_to_integer(V),
+        V
+    catch
+        _:_ ->
+            "\"" ++ V ++ "\""
+    end.
+
+session_usage() ->
+    ["vmq-admin session <sub-command>\n\n",
+     "  Retrieve information on live sessions.\n\n",
+     "  Sub-commands:\n",
+     "    list        lists all running sessions\n",
+     "  Use --help after a sub-command for more details.\n"
+    ].
+
+vmq_session_list_usage() ->
+    Options = [io_lib:format("  --~p\n", [Item])
+               || Item <- vmq_info:session_info_items()],
+    ["vmq-admin session list\n\n",
+     "  Prints some information on running sessions\n\n",
+     "Options\n\n" | Options
```

Comment:
The new `session list` cmd shows more information than the old one. There's way too much to make sense of and the column headers a truncated which is less than helpful.

```
$ _build/default/rel/vernemq/bin/vmq-admin session list
+------+------+--------+------+-----+----+----+-----+--------+------+-------+-------+----+-----+----+
|__moun|clean_|client_i|delive|is_of|is_o|is_p|mount|  node  |num_se|offline|online_|queu|sessi|stat|
+------+------+--------+------+-----+----+----+-----+--------+------+-------+-------+----+-----+----+
|      | true |mosqsub/|fanout|false|true|fals|     |VerneMQ@|  1   |   0   |   0   |<0.4|<0.45|onli|
+------+------+--------+------+-----+----+----+-----+--------+------+-------+-------+----+-----+----+
```

I'd therefore propose that we by default only quuery a subset of the information. Something like: 

```
$ _build/default/rel/vernemq/bin/vmq-admin session list --client_id --user --is_online --statename --peer_host --peer_port
+------------------+---------+---------+---------+---------+---------+
|    client_id     |is_online|peer_host|peer_port|statename|  user   |
+------------------+---------+---------+---------+---------+---------+
|mosqsub/1840-algol|  true   |localhost|  43118  | online  |undefined|
+------------------+---------+---------+---------+---------+---------+
```

I would then change the line `Prints some information on running sessions\n\n` to something like:

```
  Prints information about running sessions.
      Default options: --client_id --user --is_online --statename --peer_host --peer_port 
```